jormungandr-lib/src/interfaces/reward_parameters.rs

```diff
@@ -98,6 +128,11 @@ mod test {
                     epoch_start: Epoch::arbitrary(g),
                     epoch_rate: NonZeroU32::new(Arbitrary::arbitrary(g))
                         .unwrap_or(NonZeroU32::new(1).unwrap()),
+                    rewards_limit: Ratio::arbitrary(g),
+                    npools: NonZeroU32::new(Arbitrary::arbitrary(g))
+                        .unwrap_or(NonZeroU32::new(1000).unwrap()),
```

Comment:
why `1000` ?

Reply:
This seemed to be the number of pools that were being aimed at by the testnet.  It could be set differently.  In the mainnet, it will vary over time.  Setting it to 1000 as a test case will confirm that the mechanism is working.

Reply:
1 won't have the same effect as a higher number.  If you want to test the code, it needs to be at least 2.

Reply:
Here arbitrary is used for validity of the parameter. If you believe `1` is not a valid value, then add a better parsing instead of relying on the default derived one otherwise it means you allow such a value and it then means we are supposed to support that bad value.

Reply:
1 is a valid value.  I found it useful to differentiate the numbers to detect byte-swapping in the serialisation code.

> Any calculation with a calendar year should use 365.25 to account for leap-years; I see in a comment
> 
> `# Calculated as an annualised percentage rate (over 365 days). So 10/100 is maximum rate of 10% etc.`



> Any calculation with a calendar year should use 365.25 to account for leap-years; I see in a comment
> 
> `# Calculated as an annualised percentage rate (over 365 days). So 10/100 is maximum rate of 10% etc.`

I think this might be an integer calculation.  In this situation, the difference should be essentially noise.